requirements.txt

```diff
@@ -2,10 +2,8 @@
 # This file is autogenerated by pip-compile with Python 3.7
 # by the following command:
 #
-#    pip-compile --index-url=https://nexus.osdc.io/repository/pypi-all/simple
+#    pip-compile
```

Comment:
I have a preference for running the pip-compile the same as the previous run. I'm guessing that since it was defined in your conf file it was not included.